.circleci/config.yml

```diff
@@ -33,4 +33,4 @@ test:
           tags:
             only: /^v.*/
           branches:
-            ignore: /.*/
\ No newline at end of file
+            ignore: /.*/
```

Comment:
We need to pull in the latest changes in `master` to enable releases in the circle config

Reply:
Done 😄